### PR TITLE
feat(doc): Update the Governance Document

### PR DESCRIPTION
**Pull request by leonrayang**
### Motivation

In our past community operations, we adhered to vendor neutrality and established a steering committee. We frequently discussed matters through WeChat; however, these discussions were not formally incorporated into the governance documentation.

### Modifications

Clarify steering committee responsibilities, supplementary vendor neutral instructions, etc.

### Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Documentation Update (if none of the other choices apply)
- [ ] So on...

### Verifying this change

- [x] Make sure that the change passes the testing checks.

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

- [ ] Master
- [ ] MetaNode
- [ ] DataNode
- [ ] ObjectNode
- [ ] AuthNode
- [ ] LcNode
- [ ] Blobstore
- [ ] Client
- [ ] Cli
- [ ] SDK
- [ ] Other Tools
- [ ] Common Packages
- [ ] Dependencies
- [ ] Anything that affects deployment

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

### Review Expection

- [ ] `in-two-days`
- [x] `weeks`
- [ ] `free-time`
- [ ] `whenever`

### CODE REVIEW

**Review by xiaochunhe — APPROVED** [review]
LGTM

File: GOVERNANCE.md [hunk 1]
```diff
@@ -31,14 +37,13 @@ Making a community work requires input/effort from everyone. Maintainers should
 participate in Pull Request reviews. Maintainers are expected to respond to assigned Pull Requests
 in a *reasonable* time frame, either providing insights, or assign the Pull Requests to other
 maintainers.
-
 Every Maintainer is listed in the top-level [MAINTAINERS](MAINTAINERS.md) file.
 
 ## Committer
 
 Committer is an active contributor in the community who continuously makes contributions to the community by contributing codes, documentation, participating in community discussions, or helping community users answer questions, etc.
```
Comment (leonrayang) [inline GOVERNANCE.md:44]: 回答社区用户问题

File: GOVERNANCE.md [hunk 2]
```diff
@@ -31,14 +37,13 @@ Making a community work requires input/effort from everyone. Maintainers should
...
 maintainers.
-
 Every Maintainer is listed in the top-level [MAINTAINERS](MAINTAINERS.md) file.
 
 ## Committer
 
 Committer is an active contributor in the community who continuously makes contributions to the community by contributing codes, documentation, participating in community discussions, or helping community users answer questions, etc. 
 
-Typically, they need to have a good understanding of the project to help more community users quickly join the project. After becoming a committer, you will be responsible for reviewing relevant issues or PRs, and your opinions are also extremely important to the community.
+Typically, they need to have a good understanding of the project to help more community users quickly join the project. The Steering Committee will adjust the committer based on the contributor's activity, project progress and changes, and the balance of the field.After becoming a committer, you will be responsible for reviewing relevant issues or PRs, and your opinions are also extremely important to the community.
```
Comment (leonrayang) [inline GOVERNANCE.md:46]: 项目调整、技术方向均衡，权限考虑管控下

File: GOVERNANCE.md [hunk 3]
```diff
@@ -56,8 +61,9 @@ as part of the process.
 ## Changes in Maintainership
 
 If a Maintainer feels she/he can not fulfill the "Expectations from Maintainers", they are free to step down.
-
-Maintainers of a repository (or a directory) can lose their status by voluntarily stepping down for personal reasons, or due to inactivity.
+The Steering Committee will adjust the maintainer list based on the maintainer's activity,
+project progress and changes, and the balance of the field.When new modules are added or vendor change,
+the maintainer will also be considered to be changed.
```
Comment (leonrayang) [inline GOVERNANCE.md:66]: 表达方式修改

File: GOVERNANCE.md [hunk 4]
```diff
@@ -95,6 +100,14 @@ The PR should only be opened no earlier than 6 weeks before the end of the proje
 The PR should be kept open for no less than 4 weeks. The PR can only be merged follow the same
 voting process as in `Changes in Project Lead`.
 
+## vendor-neutrality
+
+Management and maintenance of vendor shared communication channels, such as social media and messaging platforms.
```
Comment (leonrayang) [inline GOVERNANCE.md:105]: 核对

File: GOVERNANCE.md [hunk 5]
```diff
@@ -95,6 +100,14 @@ The PR should only be opened no earlier than 6 weeks before the end of the proje
...
+
+All vendors involved in the project are encouraged to actively participate in the topic selection process for public events.
+
+Open source conferences, activities, and resources can be attended by vendors selected by the leadership team.
```
Comment (leonrayang) [inline GOVERNANCE.md:109]: resources 去掉，leadership 改为steering commit

**Review by xiaochunhe — APPROVED** [review]

**Comment by bladehliu** [comment]
+1

**Review by xiaochunhe — APPROVED** [review]

**Comment by cessory** [comment]
+1

**Review by Cloudstriff — COMMENTED** [review]
+1

**Comment by leonrayang** [comment]
+1

**Comment by Victor1319** [comment]
+1

**Comment by awzhgw** [comment]
+1

**Comment by awzhgw** [comment]
+1

**Review by awzhgw — APPROVED** [review]

**Review by awzhgw — APPROVED** [review]
+1

**Comment by shuoranliu** [comment]
+1

**Pull request merged**